01-messaging.md

```diff
@@ -108,7 +108,8 @@ messages, a `tlv_stream` is typically placed after all currently defined fields.
 
 The `type` is a varint encoded using the BigSize format. It functions as a
 message-specific, 64-bit identifier for the `tlv_record` determining how the
-contents of `value` should be decoded.
+contents of `value` should be decoded. A custom type is defined as any type
```

Comment:
Probably nitpicking but I don't find this paragraph clear enough...
What do you think of:

```
`type` identifiers below 2^16 are reserved for use in this specification.
`type` identifiers above 2^16 can be used for experiments and application-specific messages.
```

And in the sending node requirements:

```
- When defining `type` identifiers for experiments or application-specific messages:
  - MUST use `type` identifiers >= 2^16
  - SHOULD pick random `type` identifiers to avoid collision with other experiments
  - SHOULD pick odd `type` identifiers to allow normal nodes to safely ignore them
```

Reply:
Moved definition of the range into this paragraph as you suggested. I did keep the definition of the term 'custom record' to avoid referring to experiments, app-specific message, etc in multiple places.

Also rewrote it so that the range (> 2^16) is only defined in one place.

And added statement about even `type` ids, which could also be what should be enforced for the custom record.